manifest.json.template

```diff
@@ -51,11 +51,13 @@
       "repository": "libs-snapshot-local",
       "artifact": "zss-1.27.0-rc-83-20220111184458.pax"
     },
-    "org.zowe.explorer.jobs": {
-      "version": "1.0.13"
+    "org.zowe.explorer.jobs.jobs-api-package": {
+      "version": "~2.0.1",
```

Comment:
@CarsonCook you might want to omit `~`, because manifest file binaryDependencies should have static version